scripts/start.sh

```diff
@@ -0,0 +1,6 @@
+dotnet tool restore
+cd LeaderboardBackend
+dotnet ef database update
+cd ../
+docker compose up -d
```

Comment:
`dotnet ef database update` should be after `docker compose up -d` since you need the database to be up to migrate it

Reply:
Also you should be able to do `dotnet ef database update --project LeaderboardBackend` to skip having to `cd` into `LeaderboardBackend` and back

Reply:
Added in those changes